gen/calc.py

```diff
@@ -575,6 +575,16 @@ def fz_err(msg):
             validate_int_in_range(port, 1, 65535)
 
 
+def calculate_fair_sharing_excluded_resource_names(gpus_are_scarce):
+    if gpus_are_scarce == 'true':
+        return 'gpus'
+    return ''
+
+
+def calculate_filter_gpu_resources(gpus_are_scarce):
+    return gpus_are_scarce
```

Comment:
This param is equivalent to `gpus_are_scarce`, which makes it redundant. Let's remove it and use `gpus_are_scarce` directly in `dcos-config.yaml`.